lib/comgr/src/comgr-compiler.cpp

```diff
@@ -671,6 +671,21 @@ amd_comgr_status_t AMDGPUCompiler::removeTmpDirs() {
   return AMD_COMGR_STATUS_SUCCESS;
 }
 
+amd_comgr_status_t AMDGPUCompiler::executeOutOfProcessHIPCompilation(
+    llvm::ArrayRef<const char *> Args) {
+    StringRef Exec = "/opt/rocm/hip/bin/hipcc";
+    std::vector<StringRef> ArgsV;
+    ArgsV.push_back(Exec);
+    for (StringRef Arg : Args)
+      ArgsV.push_back(Arg);
+    std::vector<Optional<StringRef>> Redirects;
+    std::string ErrMsg;
```

Comment:
We should append `ErrMsg` to `AMDGPUCompiler::LogS`

Reply:
> We should append `ErrMsg` to `AMDGPUCompiler::LogS`

fixed